server/scripts/feed-sites.js

```diff
@@ -12,7 +12,7 @@ const { createRows, verifyHeaders } = require('../utils');
       const columnMap = {
         'HCAP Site ID': 'siteId',
         'Site Name': 'siteName',
-        'Early Adopter Allocation': 'earlyAdopterAllocation',
+        'Early Adopter Allocation': 'phaseOneAllocation',
```

Comment:
This is a bit funny but it does make sense to leave it IMO - I know about this discrepancy so I just need to be careful if any site creation operational requests come in before we add a feature to create new sites in the UI (at which point the feed sites script should be hopefully deprecated other than its utility in testing)